CMakeLists.txt

```diff
@@ -32,44 +32,55 @@ set(_HDRS
 
 set(_SRCS
   include/internal/bit_utils.h
-  include/internal/linux_features_aggregator.h
-  include/internal/filesystem.h
-  include/internal/hwcaps.h
-  include/internal/stack_line_reader.h
-  include/internal/string_view.h
-  src/linux_features_aggregator.c
-  src/filesystem.c
-  src/hwcaps.c
-  src/stack_line_reader.c
-  src/string_view.c
 )
 
+macro(add_linux_detection)
+  if(NOT (APPLE OR UNIX))
```

Comment:
if(NOT UNIX)
In CMake APPLE is a Unix-like system i.e. APPLE is a subset of UNIX
see: https://cmake.org/cmake/help/latest/variable/UNIX.html

Reply:
Even better